net-analyzer/python-gvm/python-gvm-1.1.0.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
```

Comment:
Please no python3.5 for new packages, it's getting removed soon.

Reply:
done